### PR TITLE
State-District wise data processing 

### PR DESCRIPTION
**Pull request by manojVivek**
Adding the district wise data that are needed for the district level maps on the front-end.

### CODE REVIEW

File: raw_data-to-state_district_wise_data.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+const rawData = require('./raw_data');
+
+console.log('Starting district wise data processing');
+const StateDistrictWiseData = rawData.raw_data.reduce((acc, row) => {
+  let stateName = row.detectedstate;
+    if(!stateName) {
+      stateName = 'Unknown';
+    }
+  if(!acc[stateName]) {
+    acc[stateName] = {state: stateName, districtData: {}};
```
Comment (JunaidBabu) [inline raw_data-to-state_district_wise_data.js]: state:stateName is redundant right?

Comment (manojVivek) [reply raw_data-to-state_district_wise_data.js]: Fixed.

File: pushthejson [hunk 2]
```diff
@@ -21,9 +21,10 @@ npm install node-fetch
 node sheet-to-json.js
 node sheet-to-json_travel_history.js
 node sheet-to-json_raw_data.js
+node raw_data-to-state_district_wise_data.js
```
[the remark this answers is not in the document]
Comment (manojVivek) [reply pushthejson]: Added a try..catch inside the script file.

**Comment by JunaidBabu** [comment]
Changed the file name to "state_district_wise.json" since it's not "raw" anymore

**Pull request merged**